lib/noticed/bulk_delivery_methods/aws_iot.rb

```diff
@@ -0,0 +1,34 @@
+require 'aws-sdk-iotdataplane'
+
+module Noticed
+  module BulkDeliveryMethods
+    class AwsIot < BulkDeliveryMethod
+      required_options :url, :credentials, :message
```

Comment:
`url, access_key_id, secret_access_key, region, topic, payload`

not sure if region is mandatory tho